src/config/src/config.rs

```diff
@@ -1290,6 +1290,12 @@ pub struct Nats {
         To update this, delete and recreate the bucket."
     )]
     pub history: i64,
+    #[env_config(
+        name = "ZO_NATS_DELIVER_POLICY",
+        default = "",
```

Comment:
Empty string will map to `all` because of the  match statement, but I think it would be better to explicitly set `all` here.

Reply:
updated